README.md

```diff
@@ -23,13 +23,25 @@ Developed for use with [Alchemy](https://github.com/dawnlabs/alchemy)
 ### Convert
 
   To and from: `jpg`, `png`, `gif`, `bmp`, `tiff`
-  
-### Merge 
+
+### Merge
 
   To: `pdf`
-  
+
   From: `jpg`, `png`, `gif`
 
+## Development
+
+Photosorcery has been created to be used with an electron application called [Alchemy](https://github.com/dawnlabs/alchemy) which is built to run on Linux, Windows as well as MacOS. This requires Photosorcery having to be built for each of those architectures.
```

Comment:
Maybe use _platforms_ or _operating systems_ here instead of _architectures_ to avoid confusion with Go conventions where the word architecture usually stands in for instruction set architecture (x64)?